chatbot3.js

```diff
@@ -0,0 +1,55 @@
+const chatbot = [
+
+{
+input: ['Hola', 'Hi', 'Oi'],
+output: ['Hello', 'Hey', 'Greetings']
+},
+{
+ input: ['What is your favourite cusine?', 'Who is your Hero?', 'Who is your role model?'],
+output: ['I am not sure.', 'There are too many to choose from.', 'I like everyone.'],
+},
+{
+ input: ['How are you?', 'How is the weather today?', 'How is Canada doing in the Olympics?'],
+output: ['Fine', 'Great', 'Not so good'],
+},
+];
+
+/*
+It should now look like this:
+
+{
+	input: ['Hello', 'Hi', 'Greetings'],
+	output: ['Hello', 'Hey', 'Greetings']
+},
+{
+*/
+
+function reply() {
+
+    let randomNumber = Math.floor((Math.random() * 3));
+    let question = document.getElementById('input').value;
+    let filterType = null;
+
+    let answer = chatbot.filter(function(item){
```

Comment:
No need to make it into a long function, this can be done in one line as you did above